README.md

```diff
@@ -1,5 +1,11 @@
-MassTransit
-===========
+# Bestseller
+
+This is a bestseller fork of MassTransit.
+Primarily used and managed by @BESTSELLER/digital-media-marketing for the [StorM project](https://backstage.k8s.bestcorp.net/catalog/default/system/storm)
+
+For now, these packages will be manually packed and pushed to the GitHub nuget repository
+
+# MassTransit
```

Comment:
I'd suggest hiding any internal teams/projects, as the repo is public. Also I think we can leverage the [Fork Sync](https://github.com/marketplace/actions/fork-sync) action to try keep up to date with upstream?

```suggestion
# BESTSELLER TECH x MassTransit
===========

This is a [BESTSELLER TECH](https://bestseller.tech/) fork of [MassTransit](https://github.com/MassTransit/MassTransit).

These packages follow upstream/master and auto-sync using the [Fork Sync GitHub Action](https://github.com/marketplace/actions/fork-sync).

# MassTransit
```

Reply:
Possibly also add a little information on the prefixing/versioning convention that will be published?

Reply:
Well, I can add the fork sync, but I'm not sure it is ideal. MassTransit package versions would not be bumped automatically. A dev would have to go and bump them automatically unless we align our packing and pushing method with what is used in the upstream repository.

Reply:
After I hit submit on the review, I thought that too. The special SemVer stuff will screw up a lot... The only way around that I considered was to keep the version as normal in the package, but then at bundle time, modify the version with our `-x.y.z` addition at that point. Maybe that could be stored as an additional value on the `csproj` too, so it can be kept side by side?

Bit more work to do it though

Reply:
The issue is that the csproj files did not contain the package version from upstream. Only the workflow sets the version.

But, I figured, let's just try and see what happens when we align with the upstream way of versioning and pushing. I've committed that now